src/phoenix/server/api/input_types/SpanSort.py

```diff
@@ -68,10 +74,22 @@ class SpanSort:
     eval_result_key: Optional[EvalResultKey] = UNSET
     dir: SortDir
 
-    def to_orm_expr(self) -> Any:
-        if self.col:
+    def __call__(self, stmt: Select[Any]) -> Select[Any]:
```

Comment:
maybe `update_sql` is better, because `sort(stmt)` seems to imply that it's actually sorting something
```suggestion
    def update_sql(self, stmt: Select[Any]) -> Select[Any]:
```

Reply:
addressed in c7f9fd55a0e925e05845f1a1eb80519ceca554be